### PR TITLE
Add process manager guide

### PR DESCRIPTION
**Pull request by lauriejim**
#### Description of what you did:

Many times we have questions about starting the app using `server.json` file.
Here is an example.

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [x] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: docs/3.0.0-beta.x/guides/process-manager.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+### Starting with strapi command
+
+By default there is two important commands.
+
+- `yarn develop` to start your project in development mode.
+- `yarn start` to start your app for production.
+
+You can also start your process manager using the `yarn start` command.
+
+`pm2 start npm --no-automation --name app -- run start`
```
Comment (derrickmehaffy) [inline docs/3.0.0-beta.x/guides/process-manager.md]: the `--no-automation` shouldn't be needed.

Comment (lauriejim) [reply docs/3.0.0-beta.x/guides/process-manager.md]: True! I was pretty sure that I deleted it.
Anyway - fixed.

**Comment by lauriejim** [comment]
I didn't understand what you mean?

File: docs/3.0.0-beta.x/guides/process-manager.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+`pm2 start npm --name app -- run start`
+
+## Configuration file
+
+PM2 let you create a config file to save all information to start your sever properly at anytime.
+
+By running `pm2 init` it will init an `ecosystem.config.js` in your application.
+
+Then replace the content of this files by the following code
```
Comment (soupette) [inline docs/3.0.0-beta.x/guides/process-manager.md]: file

**Comment by lauriejim** [comment]
About  `pm2 startup` and `pm2 save` I don't think it's necessary to mention it.
Don't want to go too much deeper. And It will depend on each process manager.

**Pull request merged**